app/models/artist_survey.rb

```diff
@@ -1,2 +1,5 @@
 class ArtistSurvey < ActiveRecord::Base
+  belongs_to :artist
+
+  # TODO does this need any validation?
```

Comment:
It should probably validate that it is there is an artist present. Often length validations make sense for string fields. Actually adding more code to this model is out of scope but the comment is valid.